tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3491,20 +3503,40 @@ <h2>Show similarity to selected datapoint</h2>
             return b.meanAbsError - a.meanAbsError;
           } else if (this.selectedFeatureSort == 'Mean squared error') {
             return b.meanSquaredError - a.meanSquaredError;
+          } else if (this.selectedFeatureSort == 'Median error') {
+            return b.medianError - a.medianError;
+          } else if (this.selectedFeatureSort == 'Median absolute error') {
+            return b.medianAbsError - a.medianAbsError;
+          } else if (this.selectedFeatureSort == 'Median squared error') {
+            return b.medianSquaredError - a.medianSquaredError;
           }
         });
       },
 
       fillInRegressionStats: function(regressionStats, name) {
         // From initial regression stats having regression error per example,
-        // calculate mean error, mean abs error, and mean squared error.
+        // calculate error statistics.
         function mean(data) {
           const sum = data.reduce((sum, value) => {
             return sum + value;
           }, 0);
 
           return sum / data.length;
         }
+        function median(sortedData) {
```

Comment:
It may be a style thing, but if we sorted arrays within the median function, we could get rid of some repetition in the inputs below (l3549-l3551).

Reply:
done